common/api/core-frontend.api.md

```diff
@@ -7497,12 +7498,14 @@ export class MapLayerSources {
 
 // @public
 export enum MapLayerSourceStatus {
-    InvalidCoordinateSystem = 6,
+    // @beta
+    IncompatibleFormat = 3,
+    InvalidCoordinateSystem = 7,
     InvalidCredentials = 1,
     InvalidFormat = 2,
-    InvalidTileTree = 3,
-    InvalidUrl = 4,
-    RequireAuth = 5,
+    InvalidTileTree = 4,

```

Comment:
Changing the numeric value of an enum member is a breaking API change, which is not ok for `@public` APIs.